apps/opendosm/pages/dashboard/balance-of-payments/index.tsx

```diff
@@ -0,0 +1,116 @@
+
+const BalanceOfPayments: Page = ({
+  last_updated,
+  meta,
+  timeseries,
+  timeseries_callout,
+  bop_snapshot,
+}: InferGetStaticPropsType<typeof getStaticProps>) => {
+  const { t } = useTranslation(["dashboard-bop", "common"]);
+  const TAB_OPTIONS: Array<OptionType> = [
+    {
+      label: t("keys.snapshot"),
+      value: "snapshot",
+    },
+    {
+      label: t("keys.timeseries"),
+      value: "timeseries",
+    },
+  ];
+
+  const { data, setData } = useData({
+    tab_index: TAB_OPTIONS[0],
+  });
+  return (
+    <AnalyticsProvider meta={meta}>
+      <Metadata title={t("header")} description={t("description")} keywords={""} />
+      <Hero
+        background="blue"
+        category={[t("common:categories.national_accounts"), "text-primary dark:text-primary-dark"]}
+        header={[t("header")]}
+        description={[t("description")]}
+        last_updated={last_updated}
+        agencyBadge={<AgencyBadge agency={meta.agency} noRedirect={true} />}
+      />
+
+      <nav className="sticky top-14 z-20 flex overflow-hidden border-b border-b-outline bg-white dark:border-b-washed-dark dark:bg-black min-[350px]:justify-center lg:static">
+        <div
+          className={
+            "hide-scrollbar max-[420px]:justify-center, flex snap-x snap-mandatory scroll-px-9 flex-nowrap overflow-x-auto max-sm:justify-start"
+          }
+        >
+          {TAB_OPTIONS.map(tab => (
+            <div key={tab.value} className="snap-start">
+              <div
+                className="flex h-full min-w-[56px] cursor-pointer items-center justify-center px-3 outline-none"
+                onClick={() => setData("tab_index", tab)}
+              >
+                <div className="relative flex h-full flex-col items-center justify-center px-2 py-4">
+                  <div
+                    className={clx(
+                      "flex items-center gap-2",
+                      data.tab_index.value === tab.value ? "text-black dark:text-white" : "text-dim"
+                    )}
+                  >
+                    <span className="whitespace-nowrap text-base font-medium">{tab.label}</span>
+                  </div>
+                  {data.tab_index.value === tab.value && (
+                    <div className="absolute bottom-0 inline-flex h-[2px] w-full min-w-[56px] rounded-full bg-primary dark:bg-primary-dark" />
+                  )}
+                </div>
+              </div>
+            </div>
+          ))}
+        </div>
+      </nav>
+
+      {data.tab_index.value === "snapshot" && (
+        <BalanceOfPaymentsSnapshot
+          last_updated={last_updated}
+          bop_snapshot={bop_snapshot}
+          meta={meta}
+        />
+      )}
+      {data.tab_index.value === "timeseries" && (
+        <BalanceOfPaymentsTimeseries
+          last_updated={last_updated}
+          timeseries={timeseries}
+          timeseries_callout={timeseries_callout}
+          meta={meta}
+        />
+      )}
```

Comment:
Not really keen with this implementation. Can you extract this to `dashboards/bop` in `layout.tsx` page? Then wrap in index page.

Also, try to be explicit with the naming of files. Some abbreviations are not common enough (like gdp) to be understood quickly

- BOP -> balance-of-payments
- IIP -> international-investment-position